docs/fides/docs/quickstart/docker.md

```diff
@@ -1,12 +1,12 @@
-# Running Fidesctl in Docker
+# Running Fidesctl with Docker Compose
 
 ---
 
-The recommended way to get Fidesctl running is via Docker. The following guide will describe how to get things going, step-by-step.
+The recommended method for running Fidesctl is via a Docker Compose environment. 
 
 ## System Requirements
 
-Docker and Docker-Compose are the only requirements here.
+Docker and Docker-Compose are all that is needed to get started.
 
 1. Install `docker` locally (see [Docker Desktop](https://www.docker.com/products/docker-desktop) or your preferred installation). The minimum verified Docker version is `20.10.8`
 1. If your `docker` installation did not include `docker-compose`, make sure to get at least version `1.29.0`. Installation instructions can be found [here](https://docs.docker.com/compose/install/).
```

Comment:
how might i know that i don't have docker compose?

Reply:
additionally, was there ambiguity to the docker setup instructions below? for example, where do I put the `docker-compose.yml` file?

Reply:
Definitely agree! Updated based on feedback.